src/main/java/core/basesyntax/service/ProcessService.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface ProcessService {
+    Storage processObjects(List<FruitTransaction> data);

```

Comment:
we can directly access Storage
```suggestion
    void processObjects(List<FruitTransaction> data);

```